k8s/tolgee.yaml

```
@@ -0,0 +1,126 @@
+# Tolgee DEV deployment. Secrets are intentionally created out of band.
+# The deployment is intentionally minimal: it reuses the existing Docker image
+# and publishes it through the shared internal DEV ingress domain.
+---
+apiVersion: v1
+apiVersion: v1
```

YAMLlint (1.37.1)

[error] duplication of key "apiVersion" in mapping

(key-duplicates)